configs/esm_software/esm_runscripts/defaults_geomar.yaml

```diff
@@ -25,17 +25,36 @@ per_model_defaults:
       input:
          all_directions: link
       restart_in:
-         # seb-wahl: need to copy restart files by default
```

Comment:
Are you using this file at all? I thought you'd be doing the changes via the `foci` setup files. If that's the case you can remove this file. If you need this file around I have not problem with keeping it.